@xen-orchestra/backups/_cleanVm.test.js

```diff
@@ -394,19 +396,19 @@ describe('tests multiple combination ', () => {
         if (useAlias) {
           const dataSurvivors = await handler.list(basePath + '/data')
           // the goal of the alias : do not move a full folder
-          expect(dataSurvivors).toContain('ancestor.vhd')
-          expect(dataSurvivors).toContain('grandchild.vhd')
-          expect(dataSurvivors).toContain('cleanAncestor.vhd')
-          expect(survivors).toContain('clean.vhd.alias.vhd')
-          expect(survivors).toContain('child.vhd.alias.vhd')
-          expect(survivors).toContain('grandchild.vhd.alias.vhd')
-          expect(survivors.length).toEqual(4) // the 3 ok + data
-          expect(dataSurvivors.length).toEqual(3) // the 3 ok + data
+          sinon.match.in(dataSurvivors, 'ancestor.vhd')
```

Comment:
Have you tested this?

I don't think that's how Sinon's matchers are meant to be used: https://sinonjs.org/releases/v14/matchers/

Reply:
Yes, tested it. The result was the same as expected.

But I'll look for an other method.